### PR TITLE
[Ceres]: Document FATE delegation signatures

### PR DESCRIPTION
**Pull request by hanssv**
In Ceres we introduce better, more structured, delegation signatures. This (a) properly documents the old delegation signature, and (b) documents the new more structured signatures.

This PR is supported by the Æternity Foundation

### CODE REVIEW

**Review by davidyuk — APPROVED** [review]
Related issues https://github.com/aeternity/aeternity/issues/4177 https://github.com/aeternity/aeternity/issues/4178

File: contracts/fate.md [hunk 1]
```diff
@@ -919,6 +919,90 @@ the RLP encoding of the RLP encoding of the empty map i.e. the bytes 130, 47, 0.
...
+has not changed from the description above, only the exact bytes to be signed
+and its structure.
+
+For general information about serialization, RLP encoding, etc, see the general
+[serialization document](../serializations.md).
+
+We use the tag `0x1a01` to identify delegation signatures. We use the following
+tags/types to identify the different delegation signatures:
+
+| Tag | Delegation type |
```
Comment (davidyuk) [inline contracts/fate.md:961]: With this tag, we don't need prefixed ids and can use binary as before, but I don't have a preference.

Comment (hanssv) [reply contracts/fate.md:961]: Let's do it like this, it is in line with the other serializations...

File: contracts/fate.md [hunk 2]
```diff
@@ -919,6 +919,90 @@ the RLP encoding of the RLP encoding of the empty map i.e. the bytes 130, 47, 0.
...
+ - `AENS_PRECLAIM` - the user signs: `owner account + contract`
+ - `AENS_CLAIM`,`AENS_UPDATE`, `AENS_TRANSFER`, `AENS_REVOKE` - the user signs:
+   `owner account + name hash + contract`
+ - AENS wildcard (valid for any name) - the user signs:
+   `owner account + contract` [New in `FATE_03`]
+ - `ORACLE_REGISTER`, `ORACLE_EXTEND` - the user signs: `owner account + contract`
+ - `ORACLE_RESPOND` - the user signs: `query id + contract`
+
+To protect about cross network re-use of signatures, the data to be signed is
+also prefixed with the _network id_.
```
Comment (radrow) [inline contracts/fate.md:944]: Why not mention the net id everywhere? When I see "the user signs" I would expect that the user signs exactly that with no qualifications in a paragraph after. 

This is what I hate in law language the most.

**Review by radrow — APPROVED** [review]

**Pull request merged**